components/servo/fake-ld.sh

```diff
@@ -1,3 +1,16 @@
-#!/bin/bash
-TARGET_DIR=$OUT_DIR/../../..
-arm-linux-androideabi-gcc $@ $LDFLAGS -lc -o $TARGET_DIR/libservo.so -shared && touch $TARGET_DIR/servo
+#!/usr/bin/env bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET_DIR="${OUT_DIR}/../../.."
+arm-linux-androideabi-gcc "$@" \
+                          "${LDFLAGS}" \
+                          -lc \
```

Comment:
Let's put the `LDFLAGS`, `lc` and `shared` flags on the same line since they're all linker arguments.